nameko/testing/waiting.py

```diff
@@ -0,0 +1,63 @@
+""" https://github.com/mattbennett/call-waiting
```

Comment:
Sorry, this was leftover. One option is to leave this code in that repo and make it a dependency. I suspect your preference is to inline it though?